### DISCUSSION TITLE
error ceate round

### DISCUSSION THREAD

**Discussion by aviejo** [evroon/bracket#212] [General]
Hello, when creating a round it always gives an error of object. Is the application 100% finished?

**Comment by evroon** [comment]
Thanks for your interest in this repo!
It's indeed not entirely finished, I will add it a warning to the README.
I will release v1.0.0 when the project is finished.

**Reply by evroon** [reply]
PR is now merged: #213

**Reply by aviejo** [reply]
Thanks. I am deploying in local windows and it is giving me cors problems part of the app. and I can't find a way to debug, I'll be attentive to your updates. maybe I can help.

**Reply by evroon** [reply]
The idea is that that should be fixed using the environment variables described here: https://evroon.github.io/bracket/docs/getting-started/configuration

But if that doesn't work or you want to disable CORS, you can also just comment these lines for now:

https://github.com/evroon/bracket/blob/master/backend/bracket/app.py#L22-L29
